src/dma/collector/workflows/readiness_check/_postgres/constants.py

```diff
@@ -0,0 +1,1651 @@
+    "seh-x-icu",
+    "seh-MZ-x-icu",
+    "ses-x-icu",
+    "ses-ML-x-icu",
+    "sg-x-icu",
+    "sg-CF-x-icu",
+    "shi-x-icu",
+    "shi-Latn-x-icu",
+    "shi-Latn-MA-x-icu",
+    "shi-Tfng-x-icu",
+    "shi-Tfng-MA-x-icu",
+    "si-x-icu",
+    "si-LK-x-icu",
+    "sk-x-icu",
+    "sk-SK-x-icu",
+    "sl-x-icu",
+    "sl-SI-x-icu",
+    "smn-x-icu",
+    "smn-FI-x-icu",
+    "sn-x-icu",
+    "sn-ZW-x-icu",
+    "so-x-icu",
+    "so-DJ-x-icu",
+    "so-ET-x-icu",
+    "so-KE-x-icu",
+    "so-SO-x-icu",
+    "sq-x-icu",
+    "sq-AL-x-icu",
+    "sq-MK-x-icu",
+    "sq-XK-x-icu",
+    "sr-x-icu",
+    "sr-Cyrl-x-icu",
+    "sr-Cyrl-BA-x-icu",
+    "sr-Cyrl-ME-x-icu",
+    "sr-Cyrl-RS-x-icu",
+    "sr-Cyrl-XK-x-icu",
+    "sr-Latn-x-icu",
+    "sr-Latn-BA-x-icu",
+    "sr-Latn-ME-x-icu",
+    "sr-Latn-RS-x-icu",
+    "sr-Latn-XK-x-icu",
+    "su-x-icu",
+    "su-Latn-x-icu",
+    "su-Latn-ID-x-icu",
+    "sv-x-icu",
+    "sv-AX-x-icu",
+    "sv-FI-x-icu",
+    "sv-SE-x-icu",
+    "sw-x-icu",
+    "sw-CD-x-icu",
+    "sw-KE-x-icu",
+    "sw-TZ-x-icu",
+    "sw-UG-x-icu",
+    "ta-x-icu",
+    "ta-IN-x-icu",
+    "ta-LK-x-icu",
+    "ta-MY-x-icu",
+    "ta-SG-x-icu",
+    "te-x-icu",
+    "te-IN-x-icu",
+    "teo-x-icu",
+    "teo-KE-x-icu",
+    "teo-UG-x-icu",
+    "tg-x-icu",
+    "tg-TJ-x-icu",
+    "th-x-icu",
+    "th-TH-x-icu",
+    "ti-x-icu",
+    "ti-ER-x-icu",
+    "ti-ET-x-icu",
+    "tk-x-icu",
+    "tk-TM-x-icu",
+    "to-x-icu",
+    "to-TO-x-icu",
+    "tr-x-icu",
+    "tr-CY-x-icu",
+    "tr-TR-x-icu",
+    "tt-x-icu",
+    "tt-RU-x-icu",
+    "twq-x-icu",
+    "twq-NE-x-icu",
+    "tzm-x-icu",
+    "tzm-MA-x-icu",
+    "ug-x-icu",
+    "ug-CN-x-icu",
+    "uk-x-icu",
+    "uk-UA-x-icu",
+    "ur-x-icu",
+    "ur-IN-x-icu",
+    "ur-PK-x-icu",
+    "uz-x-icu",
+    "uz-Arab-x-icu",
+    "uz-Arab-AF-x-icu",
+    "uz-Cyrl-x-icu",
+    "uz-Cyrl-UZ-x-icu",
+    "uz-Latn-x-icu",
+    "uz-Latn-UZ-x-icu",
+    "vai-x-icu",
+    "vai-Latn-x-icu",
+    "vai-Latn-LR-x-icu",
+    "vai-Vaii-x-icu",
+    "vai-Vaii-LR-x-icu",
+    "vi-x-icu",
+    "vi-VN-x-icu",
+    "vun-x-icu",
+    "vun-TZ-x-icu",
+    "wae-x-icu",
+    "wae-CH-x-icu",
+    "wo-x-icu",
+    "wo-SN-x-icu",
+    "xh-x-icu",
+    "xh-ZA-x-icu",
+    "xog-x-icu",
+    "xog-UG-x-icu",
+    "yav-x-icu",
+    "yav-CM-x-icu",
+    "yi-x-icu",
+    "yi-001-x-icu",
+    "yo-x-icu",
+    "yo-BJ-x-icu",
+    "yo-NG-x-icu",
+    "yrl-x-icu",
+    "yrl-BR-x-icu",
+    "yrl-CO-x-icu",
+    "yrl-VE-x-icu",
+    "yue-x-icu",
+    "yue-Hans-x-icu",
+    "yue-Hans-CN-x-icu",
+    "yue-Hant-x-icu",
+    "yue-Hant-HK-x-icu",
+    "zgh-x-icu",
+    "zgh-MA-x-icu",
+    "zh-x-icu",
+    "zh-Hans-x-icu",
+    "zh-Hans-CN-x-icu",
+    "zh-Hans-HK-x-icu",
+    "zh-Hans-MO-x-icu",
+    "zh-Hans-SG-x-icu",
+    "zh-Hant-x-icu",
+    "zh-Hant-HK-x-icu",
+    "zh-Hant-MO-x-icu",
+    "zh-Hant-TW-x-icu",
+    "zu-x-icu",
+    "zu-ZA-x-icu",
+}
+CLOUDSQL_SUPPORTED_COLLATIONS: Final[set[str]] = {
```

Comment:
Are these values taken from the cloudsql instance?

Reply:
I copied the values from alloydb for the initial implementation. 

I was going to do the same for the supported extensions list.